src/sl.json

```diff
@@ -54,7 +54,7 @@
   "hours-played": "{{count}} igrana ura",
   "hours-played_plural": "{{count}} ur igranih",
```

Comment:
Hey, if you don't mind, I'd like to test something here:

The translation software we use offers pluralization rules for many languages. In English, you just have one “plural”: `1 hour played`, `5 hours played`. But I know that Slovenian has a _dual_ on top of a plural, and in fact the software says that instead of `hours-played` and `hours-played_plural`, we should supply _four_ different translations:

* `hours-played_0` for when it's 0, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14… hours played.
* `hours-played_1` for when it's 1 hour played.
* `hours-played_2` for when it's 2 hours played.
* `hours-played_3` for when it's 3 or 4 hours played. 

I don't speak Slovenian, but I found an example online (using some different names for these groups:)

```
sl:
  apartment:
    one: "%{count} stanovanje"
    two: "%{count} stanovanji"
    few: "%{count} stanovanja"
    other: "%{count} stanovanj"
```

Would you like to do the same for `sl.json`? Wherever you find `key` and `key_plural`, replace it with

```
  "key_0": "(translation for plurals)",
  "key_1": "(translation for 1)",
  "key_2": "(translation for 2)",
  "key_3": "(translation for 3 and 4)",
```

Reply:
Hi

Could you please here at lines 54 and 55 make these changes in keys. The translations you can just fill them with "TODO" or some other placeholder and I will translate it. It's just so that I know that I'm doing things right.

Thanks :D

Reply:
I have added that. I hope it is correct 😄